.gitignore

```diff
@@ -0,0 +1,7 @@
+.tox/
```

Comment:
I miss a standard for this file on our charms. What do you think about using https://github.com/github/gitignore/blob/main/Python.gitignore as a base?

Reply:
Minor comments